index.js

```diff
@@ -53,35 +52,34 @@ app.use(express.json());
 // IDEAS METHODS
 
 app.get('/ideas', (req, res) => {
-    let sortedIdeasList;
+    let sortedIdeas = getIdeasUtil();
+    const ideas = getIdeas();
```

Comment:
Do `getIdeasUtil` and `getIdeas` return the same value? If that's the case, then can you simplify this?

Reply:
Yes, sure, you are absolutely right!  Fixed.